content/components/avatar-stack.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Avatar stack
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=13007%3A40428
+react: https://primer.style/react/AvatarStack
+description: AvatarStack displays two or more avatars in an inline stack
+---
+
+import {Box, Text} from '@primer/react'
+
+<img
+  width="960"
+  alt="A diagram of three avatars stacked next to each other."
+  src="https://user-images.githubusercontent.com/586552/203208345-b110f45e-f7e5-4fab-b972-f5d31e809f22.png"
+/>
+
+## Usage
+
+Stacked avatars can be used to show multiple collaborators or participants when there is limited space available. When you hover over the stack, the avatars will reveal themselves.
+
+### Best practices
+
+- Use AvatarStack when space is limited
+- Don’t use AvatarStack if there is adequate space to show 4 avatars or less. 
+
+## Anatomy
+
+Avatar stack displays a minimum of 2 avatars and a maximum of 4 avatars.
+Only the 20px avatar size is currently used in the avatar stack.
+
+## Options
+
+### Alignment
+
+The default AvatarStack is left-aligned. You can [right-align](https://primer.style/react/AvatarStack#right-aligned) the component for layouts that are better suited for it.
+
+## Related components
+
+- [Avatar](../avatar.mdx)
+- [Avatar pair](../avatar-pair.mdx)
```

Comment:
I don't think these links to MDX files are intentional. If this was a mistake, just update these links to be component docs pages.